docs/reference/content/reference/management/logging.md

```diff
@@ -85,19 +95,23 @@ Driver classes available for filtering:
 You can add your own classes to the logger by creating your own logger instances. 
 
 ```js
-var Logger = require('mongodb').Logger
-  , assert = require('assert');
+const Logger = require('mongodb').Logger;
+const assert = require('assert');
 
-var A = function() {
-  var logger = Logger('A', options);
+class A {
+  constructor() {
+    this.logger = Logger('A');
```

Comment:
I think this is actually required to be `new Logger('A')` (see: https://github.com/mongodb-js/mongodb-core/blob/3.0.0/lib/connection/logger.js#L25)